### PR TITLE
TAL-7: Create Puppy Tests

### PR DESCRIPTION
**Pull request by Manuyafox**
Created 3 test for wrong credentials and 1 broken test for check all is displayed (this was not part of my plans)

### CODE REVIEW

File: src/main/java/org/example/utils/WaitUtils.java [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 public class WaitUtils {
 
+    private WaitUtils() {
+    }
+
```
Comment (ElKordel) [inline src/main/java/org/example/utils/WaitUtils.java:7]: Is this necessary?

**Review by ElKordel — COMMENTED** [review]
think it's ok

File: src/main/java/org/example/utils/WaitUtils.java (hunk 1 above)
Comment (Manuyafox) [reply src/main/java/org/example/utils/WaitUtils.java:7]: Андрей просил добавить приватный конструктор

**Review by iiddd — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/utils/WaitUtils.java (hunk 1 above)
Comment (Manuyafox) [reply src/main/java/org/example/utils/WaitUtils.java:7]: This is a private constructor that Andrey asked to be added

**Review by iiddd — APPROVED** [review]

**Pull request merged**